Tests/SonarScanner.MSBuild.PreProcessor.Test/PreprocessorObjectFactoryTests.cs

```diff
@@ -63,6 +63,18 @@ public async Task CreateSonarWebService_RequestServerVersionFailedDueToGenericEx
             logger.AssertNoErrorsLogged();
         }
 
+        [TestMethod]
+        public async Task CreateSonarWebService_InvalidHostUrl_ReturnNullAndLogErrors()
+        {
+            var sut = new PreprocessorObjectFactory(logger);
+
+            var result = await sut.CreateSonarWebServer(CreateValidArguments("http:/myhost:222"), Mock.Of<IDownloader>());
+
+            result.Should().BeNull();
+            logger.AssertSingleErrorExists("The given host url 'http:/myhost:222' is an invalid URL.");
```

Comment:
What do you think about: 
"The value provided for the host URL parameter (`{0}`) is not valid. Please make sure that you have entered a valid URL and try again."

Reply:
Better wording 👍